contest/static/contest/js/contest.js

```diff
@@ -63,7 +63,7 @@ function checkTime(i) {
 function getRestTime() {
```

Comment:
This function is quite same with that in [this file](https://github.com/Changron/NTHUOJ_web/blob/dev/index/static/js/contestTimer.js)
Could you please reuse it?